Sources/Flatten.swift

```diff
@@ -481,46 +481,32 @@ extension UnsafeAtomicState where State == ProducerState {
 }
 
 extension Signal {
-	/// Merges the given signals into a single `Signal` that will emit all
-	/// values from each of them, and complete when all of them have completed.
-	///
-	/// - parameters:
-	///   - signals: A sequence of signals to merge.
+	/// Merges the values of all the given signals, in the manner described
+	/// by `merge(with:)`.
```

Comment:
Normally I wouldn't mind this, but Xcode does not make it easy to find this documentation, so I'd keep it duplicated.

Reply:
hmm, I was according to `zip`, `combineLatest` operator, though should revert?

Reply:
We should duplicate it in those too IMO (at least until SourceKit & CMD+Click actually works consistently in Xcode). For now can you keep these how they were?

Reply:
gotcha, will revert them:+1:
Write the doc for `zip` and` combineLatest`, will too much lines.
I think there are pros and cons, so not to write now.